src/components/interfaces/HMI_API.xml

```diff
@@ -2857,6 +2863,9 @@
   <param name="numCustomPresetsAvailable" type="Integer" minvalue="1" maxvalue="100" mandatory="false">
     <description>The number of on-screen custom presets available (if any); otherwise omitted.</description>
   </param>
+  <param name="menuLayoutsAvailable" type="Common.MenuLayout" array="true" minsize="1" maxsize="1000" mandatory="false">
```

Comment:
```suggestion
  <param name="menuLayoutsAvailable" type="Common.MenuLayout" array="true" minsize="1" maxsize="1000" mandatory="false" since="6.0">
```